pyresample/geometry.py

```diff
@@ -1050,8 +1078,10 @@ def _compute_bound_centers_dask(self, proj_dict, lons, lats):
         y_is_pole = (ymax >= 90 - epsilon) or (ymin <= -90 + epsilon)
         if crs.is_geographic and x_passes_antimeridian and not y_is_pole:
```

Comment:
If the crs is geographic, I would rather go to cartesian coordinates to do the computations here... but maybe this should be another PR?

Reply:
Yes probably another PR. However, I'm not sure why cartesian coordinates would work better for this math. It is meant to be a simple workaround so my hope was that it wouldn't get too complicated. But if you see something then of course we could do it.

Reply:
it's just that I don't like the thresholding needed around poles and antimeridian...